README.md

```diff
@@ -90,4 +96,14 @@ To use email notification functionality you must specify mailer configuration in
 mailer:
   sender: do-not-reply@example.com
   server: smtp.example.com
+```
+
+## IRC notification
+
+To use irc notification functionality you must specify irc server configuration in config file
+```
+irc:
+  server: irc.example.com
+  port: 12345
+  channel: "#example"
```

Comment:
inconsistency of arguments with examples/sampleinput_irc.yaml

Reply:
changed